server/pulp/server/managers/content/upload.py

```diff
@@ -204,8 +205,11 @@ def import_uploaded_unit(repo_id, unit_type_id, unit_key, unit_metadata, upload_
 
         # Invoke the importer
         try:
-            return importer_instance.upload_unit(transfer_repo, unit_type_id, unit_key,
-                                                 unit_metadata, file_path, conduit, call_config)
+            result = importer_instance.upload_unit(transfer_repo, unit_type_id, unit_key,
+                                                   unit_metadata, file_path, conduit, call_config)
+            repo_controller.update_unit_count(repo_obj, unit_type_id, 1)
```

Comment:
Let's chat about the case where the unit already existed in the repo. I'm not sure if all of the importers would raise an error in that case. If any of them silently let that slide in the name of idempotency, we could end up incorrectly incrementing.